doc/classes/Control.xml

```diff
@@ -117,6 +117,16 @@
 			<argument index="1" name="data" type="Variant">
 			</argument>
 			<description>
+				Godot will call this method on your behalf to ask if [code]data[/code] from a Control's [method get_drag_data] can be dropped at [code]position[/code]. Position is local to this Control.
```

Comment:
- Try to stick to present tense rather than future
- I don't think "on your behalf" is necessary
- "to ask if" is misleading
- Don't capitalize "Control" when it's being used generically, only when referring to the actual class itself.

Godot calls this method to test if [code]data[/code] from a control's [method get_drag_data] can be dropped at [code]position[/code]. [code]position[/code] is local to the control.